### PR TITLE
feat(language): enable google auto detection

### PR DESCRIPTION
**Pull request by redbayoub**
if the language attribute is not set allow google to auto detect the user browser language
[recaptcha docs](https://developers.google.com/recaptcha/docs/display#javascript_resource_apijs_parameters)

### CODE REVIEW

**Review by farnabaz — APPROVED** [review]
Thank you for the contribution 👍

File: lib/plugin.js [hunk 1]
```diff
@@ -111,7 +111,10 @@ class ReCaptcha {
     script.setAttribute('defer', '')
 
     if (this.version === 2) {
-      script.setAttribute('src', API_URL + '?hl=' + this.language)
+      if(this.language)
```
Comment (Atinux) [inline lib/plugin.js:114]:
```suggestion
      if (this.language)
```

**Comment by Atinux** [comment]
Does this become a breaking change?

**Comment by farnabaz** [comment]
This will change language for non-english websites. So yes it will break the current behavior.  
I believe this should be the default behavior of language option. Users can customize language in config.

**Pull request merged**